grr/server/grr_response_server/flows/general/osquery.py

```diff
@@ -4,16 +4,31 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import unicode_literals
+from typing import Iterator
+from typing import List
+from typing import Dict
 
-from grr_response_core.lib.rdfvalues import osquery as rdf_osquery
 from grr_response_core.lib.util import compatibility
+from grr_response_core.lib.rdfvalues import paths as rdf_paths
+from grr_response_core.lib.rdfvalues import client_fs as rdf_client_fs
+from grr_response_core.lib.rdfvalues import osquery as rdf_osquery
+from grr_response_core.lib.rdfvalues import client_action as rdf_client_action
+from grr_response_server.rdfvalues import flow_objects as rdf_flow_objects
+from grr_response_server.rdfvalues import objects as rdf_objects
+from grr_response_server.flows.general import transfer
 from grr_response_server import flow_base
 from grr_response_server import flow_responses
 from grr_response_server import server_stubs
+from grr_response_server.databases import db
 
 
 TRUNCATED_ROW_COUNT = 10
 
+FILE_COLLECT_LIMIT_MAX_COLUMNS = 5
+FILE_COLLECT_LIMIT_MAX_ROWS = 1000
+FILE_COLLECT_LIMIT_MAX_SINGLE_FILE_BYTES = 2**30 // 2  # 1/2 GiB
+FILE_COLLECT_LIMIT_MAX_TOTAL_BYTES = 2**30  # 1 GiB
```

Comment:
Having both `MAX` and `LIMIT` feels kind of redundant. Why not just `FILLE_COLLECTION_COLUMN_LIMIT` etc.?

Reply:
Yes I agree it would be better to leave one. It's a small thing, but isn't it better to preserve `MAX` since technically we could `LIMIT` both the minimum or the maximum?